setup.py

```diff
@@ -25,7 +25,7 @@
         'Framework :: Django',
         'Topic :: Internet :: WWW/HTTP :: Browsers',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.10',
```

Comment:
See above, if PYTHON_VERSION is specified in the container, you can substitute here, maybe it is defined in settings? or straight up environment.

Reply:
Well since we're not publishing the package anyway I'm thinking of completely removing this classifier actually. It shouldn't matter.